src/Parser/Priest/Shadow/Modules/Items/Tier21_4set.js

```diff
@@ -28,17 +29,24 @@ class Tier21_4set extends Analyzer {
     const spellId = event.ability.guid;
     const combatant = this.combatants.selected;
 
-    if (spellId !== SPELLS.MIND_BLAST.id && spellId !== SPELLS.MIND_FLAY.id) return;
+    if (spellId !== SPELLS.MIND_BLAST.id && spellId !== SPELLS.MIND_FLAY.id && spellId !== SPELLS.VOID_BOLT.id ) return;
     if (!combatant.hasBuff(SPELLS.VOIDFORM_BUFF.id)) return;
 
     const { stacks: currentVoidformStacks } = combatant.getBuff(SPELLS.VOIDFORM_BUFF.id);
     const increasedCritChance = currentVoidformStacks * SET_INCREASE_CRIT_CHANCE_PER_VOIDFORM_STACK / 100;
 
     const critChanceWithoutSet = 1 + combatant.critPercentage;
     const critChanceWithSet = increasedCritChance + critChanceWithoutSet;
-
-    const hitDamage = event.hitType === HIT_TYPES.CRIT ? (calculateEffectiveDamage(event, 1) / CRIT_TO_HIT_MODIFIER) : (calculateEffectiveDamage(event, 1));
-
+    var hitDamage = 0.0;
```

Comment:
use `let` instead of `var`, then travis should pass (it should also tell you this when you run it)